core/src/test/java/org/elasticsearch/indices/IndicesServiceTests.java

```diff
@@ -187,4 +199,54 @@ public void testPendingTasks() throws Exception {
         assertAcked(client().admin().indices().prepareOpen("test"));
 
     }
+
+    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/17695")
+    public void testDeletingSameIndexDirectoryFromConcurrentProcesses() throws Exception {
```

Comment:
Makes sense, I'll remove it, then merge this branch.  Thanks for the review @bleskes !